### PR TITLE
CLOUDSTACK-10360: Change the method name.

### PR DESCRIPTION
**Pull request by BruceKuiLiu**
The method is named as "scoped" that seems to whether the variable config is scoped in _scopedStorages or not.
Actually, the method tries to find a storage of which scope equals to the scope of config.
So that, the method name "findStorage" should be more clear than "scoped".

### CODE REVIEW

File: framework/config/src/main/java/org/apache/cloudstack/framework/config/impl/ConfigDepotImpl.java [hunk 1]
```diff
@@ -166,7 +166,7 @@ public ConfigurationDao global() {
         return _configDao;
     }
 
-    public ScopedConfigStorage scoped(ConfigKey<?> config) {
+    public ScopedConfigStorage findStorage(ConfigKey<?> config) {
```
Comment (rafaelweingartner) [inline framework/config/src/main/java/org/apache/cloudstack/framework/config/impl/ConfigDepotImpl.java]: I would use something like `findScopedConfigStorage`. This `findStorage` can be confused with find storage pool.

Comment (BruceKuiLiu) [reply framework/config/src/main/java/org/apache/cloudstack/framework/config/impl/ConfigDepotImpl.java]: Thanks, "findScopedConfigStorage" is more clear.

**Comment by DaanHoogland** [comment]
I like your take on radable code @BruceKuiLiu though I didn't agree with all your PRs so far, please keep it up

**Comment by DaanHoogland** [comment]
@rhtyd @rafaelweingartner @wido @mike-tutkowski @anyone, do we need testing on this change?

**Pull request merged**